core/src/main/java/com/oath/oak/InternalOakMap.java

```diff
@@ -104,7 +103,11 @@ void close() {
         // once reference count is zeroed, the map meant to be deleted and should not be used.
         // reference count will never grow again
         if (res == 0) {
-            memoryManager.close();
+            try {
+                memoryManager.close();
+            } catch (IOException e) {
```

Comment:
Why/what exception can happen here?

Reply:
When stating that MemoryManager implements Closable, it told me to add this try-catch clause.